lib/metaData.js

```diff
@@ -50,9 +59,29 @@ module.exports = async (url, model = null) => {
                 Logger.error("Edmx error:", error.message);
                 throw error;
             }
+            break;
+        case COMPILER_TYPES.mcp:
+            try {
+                if (isMCPPluginAvailable()) {
+                    const { buildMcpServerDefinitionByService } = require("@btp-ai/mcp-plugin");
+                    // Get all available CDS services
+                    const allServices = Object.values(cds.services);
+                    // Generate metadata from runtime services
+                    responseFile = await buildMcpServerDefinitionByService(allServices);
```

Comment:
Should we keep this function? I don't see why not, it's more flexible - in the future we could decide to split MCP servers by CDS Service - and for now we don't.